presentation/frontend/package.json

```diff
@@ -35,8 +38,9 @@
     "extract": "betools extract --fileExt=ts --extractFrom=./src/test --recursive --out=../../generated-docs/extract",
     "extract-api": "betools extract-api --entry=presentation-frontend",
     "lint": "eslint -f visualstudio \"./src/**/*.ts\" 1>&2",
-    "pseudolocalize": "betools pseudolocalize --englishDir ./lib/public/locales/en --out ./lib/public/locales/en-PSEUDO",
-    "test": "mocha -r jsdom-global/register --config ../.mocharc.json \"./lib/test/**/*.test.js\"",
+    "pseudolocalize:cjs": "betools pseudolocalize --englishDir ./cjs/public/locales/en --out ./cjs/public/locales/en-PSEUDO",
```

Comment:
Pseudolocalization used to be part of `build` script - now it's not. Also, it seems the paths aren't correct.

Reply:
Frotnend always copied localization files from common/assets into here, but removed this, and therefore this isnt required anymore, removing